e2e/steps_file.js

```diff
@@ -73,5 +74,10 @@ module.exports = function () {
       const href = `${baseUrl}/case/${config.definition.jurisdiction}/${config.definition.caseType}/${caseId.replace(/\D/g, '')}`;
       this.navigateToUrl(href);
     },
+
+    seeSubmissionDate(row){
+      let currentDate = new Date();
```

Comment:
Rebecca since you not reassigning it in the future it makes sense to use `const` keyword instead of `let`. Just suggestion for the future, I do not require making that change here now.

Reply:
removed as test removed